plugin/storage/es/spanstore/schema.go

```diff
@@ -0,0 +1,183 @@
+               "type":"long"
+            },
+            "flags":{
+               "type":"integer"
+            },
+            "logs":{
+               "properties":{
+                  "timestamp":{
+                     "type":"long"
+                  },
+                  "tags":{
+                     "type":"nested",
+                     "dynamic":false,
+                     "properties":{
+                        "key":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "value":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "tagType":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        }
+                     }
+                  }
+               }
+            },
+            "process":{
+               "properties":{
+                  "serviceName":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "tags":{
+                     "type":"nested",
+                     "dynamic":false,
+                     "properties":{
+                        "key":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "value":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "tagType":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        }
+                     }
+                  }
+               }
+            },
+            "references":{
+               "type":"nested",
+               "dynamic":false,
+               "properties":{
+                  "refType":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "traceID":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "spanID":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  }
+               }
+            },
+            "tags":{
+               "type":"nested",
+               "dynamic":false,
+               "properties":{
+                  "key":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "value":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "tagType":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  }
+               }
+            }
+         }
+      },
+      "service":{
```

Comment:
I think it makes sense to separate the index for future proofing. Let's add a TODO and punt on it for now until we get a prototype working for v5. I don't think it's gonna be a big change to separate out the indices.